NFC_EEPROM/source/target/TARGET_ST25DV/eeprom_driver/ST25DV/st25dv.cpp

```diff
@@ -0,0 +1,2403 @@
+/**
+  ******************************************************************************
+  * @file    st25dv.c 
+  * @author  MMY Application Team
+  * @version $Revision: 3308 $
+  * @date    $Date: 2017-01-13 11:19:33 +0100 (Fri, 13 Jan 2017) $
+  * @brief   This file provides set of driver functions to manage communication 
+  *          between BSP and ST25DV chip.
+  ******************************************************************************
+  * @attention
+  *
+  * <h2><center>&copy; COPYRIGHT 2016 STMicroelectronics</center></h2>
+  *
+  * Licensed under ST MYLIBERTY SOFTWARE LICENSE AGREEMENT (the "License");
```

Comment:
I don't think this license is compatible with Mbed OS - would it be possible to relicense under Apache 2? Additionally I believe this driver should live in its own repo or within Mbed OS itself as it can be useful to use elsewhere.